win32/customactions.cpp

```diff
@@ -206,6 +206,10 @@ MD_REGISTRATION minidriver_registration[] = {
                                           14, {0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF}},
     {TEXT("Swissbit iShield Key Pro CL"),     {0x3b,0x87,0x80,0x01,0x69,0x53,0x68,0x69,0x65,0x6c,0x64,0x50},
                                           12, {0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF}},
+	{TEXT("Swissbit iShield Key 2"),      {0x3b,0xd5,0x18,0xff,0x81,0xb1,0xfe,0x45,0x1f,0xc3,0x80,0x73,0xc8,0x21,0x10,0x6f},
+	                                      16, {0xFF,0xFF,0xFF,0xFF,0xFF,0xFF, 0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF}},
+	{TEXT("Swissbit iShield Key 2 CL"),   {0x3b,0x85,0x80,0x01,0x80,0x73,0xc8,0x21,0x10,0x0e},
+	                                      10, {0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF}},
```

Comment:
Could you please fix indentation?

Reply:
sure, fixed

Reply:
now it is fixed by replacing `tab` with `4xspace`